src/js/views/patients/patient/data-events/data-events_views.js

```diff
@@ -181,10 +206,11 @@ const ListView = CollectionView.extend({
   },
   emptyView: EmptyView,
   viewComparator({ model }) {
-    return - model.moment('updated_at').format('x');
+    const comparatorAttr = model.get('finished_at') ? 'finished_at' : 'updated_at';
```

Comment:
I believe this can just be `return model.get('updated_at') || model.get('finished_at');`  _assuming_ that `finished_at` is the data format we've been using... if that's the case our default date-type sorts alphabetically and is way faster than using moment.

Reply:
I think we're going to need a slightly more complex sort comparator since we need to display most recent date first.